pkg/cli/admin/upgrade/status/examples/to-multi-arch.detailed-output

```diff
@@ -10,18 +10,22 @@ Updating Cluster Operators
 NAME             SINCE   REASON   MESSAGE
 machine-config   6m5s    -        Working towards 4.18.0-ec.3
 
-All control plane nodes successfully updated to 4.18.0-ec.3
+Control Plane Nodes
+NAME                                        ASSESSMENT    PHASE      VERSION       EST   MESSAGE
+ip-10-0-33-81.us-east-2.compute.internal    Progressing   Updating   4.18.0-ec.3   +5m   
```

Comment:
Feels like the `EST=+5m` could bring back `Est. Time Remaining` that we lost [here](https://github.com/openshift/oc/pull/1926/files#diff-c7e8bffaea9a52c9437d3d7359cfad687ffeda2b0750e96ad12fde44eca00689R466-R468).

Reply:
Unfortunately they are not quite connected at the moment. The `EST` here are basically hardcoded values for a given PHASE, `Est. Time. Remaining` works differently.